gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -360,8 +360,8 @@ if CLIENT then
 
 		-- draw crosshair dot
 		if cvCrosshairMode:GetInt() == CROSSHAIR_MODE_DOT_AND_LINES or cvCrosshairMode:GetInt() == CROSSHAIR_MODE_DOT_ONLY then
-			local xDot = mathFloor(xCenter - thicknessLine * 0.5)
-			local yDot = mathFloor(yCenter - thicknessLine * 0.5)
+			local xDot = xCenter - mathFloor(thicknessLine * 0.5)
+			local yDot = yCenter - mathFloor(thicknessLine * 0.5)
```

Comment:
Just use the offsetLine variable that has these values already calculated.

Reply:
```suggestion
			local xDot = xCenter - offsetLine
			local yDot = yCenter - offsetLine
```
should just be able to click this to make that change afaik